src/main/java/com/jcabi/github/User.java

```diff
@@ -244,6 +275,58 @@ public UserEmails emails() {
             return this.user.emails();
         }
 
+        /**
+         * Returns this user's notifications.
+         * @todo #913:30min Implement the call "List your notifications" as
+         *  described at https://developer.github.com/v3/activity/notifications/
+         * @return Notifications of the user
+         * @throws IOException Thrown, if an error during sending request and/or
+         *  receiving response occurs.
+         */
+        @Override
+        public List<Notification> notifications() throws IOException {
+            final String pwd = "";
+            final Response resp =
+                new ApacheRequest("https://api.github.com/notifications")
+                    .header(
+                        HttpHeaders.AUTHORIZATION,
+                        String.format(
+                            "Basic %s",
+                            DatatypeConverter.printBase64Binary(
+                                String.format(
+                                    "%s:%s",
+                                    this.user,
+                                    pwd
+                            ).getBytes(Charsets.UTF_8)
+                        )
+                    )
+                ).fetch();
+            final JsonResponse jsonresp = new JsonResponse(resp);
+            final JsonArray jsnnotifs = jsonresp.json().readArray();
+            final List<Notification> notifs =
+                new LinkedList<Notification>();
+            for (final JsonValue jsnnotif : jsnnotifs) {
+                final JsonObject notifobj = (JsonObject) jsnnotif;
+                final Notification notif = new RtNotification(
+                    notifobj.getInt("id")
+                );
+                notifs.add(notif);
+            }
+            return notifs;
+        }
+
+        /**
+         * Marks notifications of a user as read.
```

Comment:
Please remove javadoc from overriden method.
If you need to have todo you can add it like this:
``
// @todo #123:30min Description
//  More Description.
//  Even more.
``